diff --git a/dist/index.js b/dist/index.js
index 702f0e272d665fdc6cd1dd2c98f4c06cf9506f76..761565b53dd2c0bfd8bac81e840b47c0bf4f1201 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -198,3 +198,7 @@ export * from './util/compilable.js';
 export * from './util/log.js';
 export { logOnce } from './util/log-once.js';
 export { createQueryId } from './util/query-id.js';
+
+// Re-export migration constants moved to kysely/migration in 0.29.x
+export { DEFAULT_MIGRATION_TABLE, DEFAULT_MIGRATION_LOCK_TABLE } from './migration/migrator.js';
+export { DefaultQueryCompiler } from './query-compiler/default-query-compiler.js';
